packages/block-library/src/query-pagination/style.scss

```diff
@@ -32,7 +32,17 @@ $pagination-margin: 0.5em;
 		}
 	}
 
-	&.aligncenter {
-		justify-content: center;
```

Comment:
This shouldn't be removed. It was added here: https://github.com/WordPress/gutenberg/pull/34739 as a fix for `align center`.

Reply:
That behavior is achieved using the centered justification:
![image](https://user-images.githubusercontent.com/1310626/144631371-2a221e9c-3e7c-4e5b-b1f3-8c03ce9dda02.png)

Reply:
I know, but we also support the `center` alignment and is needed for backwards compatibility.

Reply:
@ntsekouras could you provide more context about backward compatibility related to this?

Reply:
Sure. Even if the `align:center` achieves the same think with centered justification, we still support this alignment. So it cannot be removed because it has created a regression. You can check this on trunk by selecting `align:center` and checking the front-end.

Even if we removed the `align:center` option though and with a new block migration in place, we would still need to keep the css rule because the migrations run and update the blocks markup in the editor and then we have to also save to get the new markup in the post content. So for content with blocks that had `align:center` but the migration was never applied, the content should look as before in the front end without and broken styles.

Makes sense? Please tell me to clarify anything  from the above if you want.